flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -63,6 +73,14 @@ class FlxGamepad implements IFlxDestroyable
 	 * Helper class to get the justMoved, justReleased, and float values of analog input.
 	 */
 	public var analog(default, null):FlxGamepadAnalogList;
+	/**
+	 * Helper class to get the float values of motion-sensing input, if it is available
+	 */
+	public var motion(default, null):FlxGamepadMotionValueList;
+	/**
+	 * Helper class to get the float values of mouse-like pointer nput, if it is available
+	 */
+	public var pointer(default, null):FlxGamepadPointerValueList;
```

Comment:
Hmm, actually looking at it I'm not sure. As for right now I'm just spitting out the raw analog output from the controller, and there's a whole bunch of uses for it. I think ONE of those uses is definitely to map that to the screen coordinates (which seems to be FlxPointer's purpose), but the game designer / player could also just want the raw analog input for something else. 

I'd love the option to map it to the screen but I don't want to over-process the data before the user has access to it.